tailwind.config.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
     }),
     screens: {
       xsm: "360px",
+      "breakpoint-500": "500px",
```

Comment:
```suggestion
      "xmd": "500px",
```

Reply:
I think we should keep it as "breakpoint-500" or maybe "bp-500" or "bpt-500"
Sometimes I forget what the exact values of `md` and `lg` are and having one more `xmd` will be painful as in I'll have to go and check again and again in the config file.
Plus it also restricts us. If we `bp-500` we could also do `bp-550` or whatever value we like!

Reply:
And for that if we have to change the above `xsm` I really don't mind. It's just what I came up with at that time. Even that is a bit wrong. In tailwind, extra small is usually referred to as `xs` instead of `xsm`

Reply:
let's go with bp